docs/src/rules/no-unused-expressions.md

```diff
@@ -79,8 +79,6 @@ Examples of **correct** code for the default `{ "allowShortCircuit": false, "all
 
 {} // In this context, this is a block statement, not an object literal
 
-{myLabel: someVar} // In this context, this is a block statement with a label and expression, not an object literal
```

Comment:
I think this a useful example, Maybe we could just fix it, for example:

```js
{ myLabel: foo() } // In this context, this is a block statement with a label and expression, not an object literal
```

Reply:
Done!
it too was not sure about this change.